src/style.css

```diff
@@ -1 +1,30 @@
 /* add styles here */
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Remove margin-right from the last stars in a row using :last-child rule

Reply:
done